apps/core/app/[locale]/(default)/(faceted)/category/[slug]/page-data.ts

```diff
@@ -0,0 +1,39 @@
+import { cache } from 'react';
+
+import { getSessionCustomerId } from '~/auth';
+import { client } from '~/client';
+import { graphql, VariablesOf } from '~/client/graphql';
+import { revalidate } from '~/client/revalidate-target';
+import { BreadcrumbsFragment } from '~/components/breadcrumbs';
+
+import { CategoryTreeFragment } from './_components/sub-categories';
+
+const PageQuery = graphql(
+  `
+    query PageQuery($categoryId: Int!) {
```

Comment:
Can we add a more specific query name? If we use a generic name it will be harder to track with our gql logger.